.github/workflows/consume-release.yml

```diff
@@ -6,13 +6,40 @@ on:
   workflow_run:
     workflows: [Release]
     types: [completed]
+    branches: [develop]
 
 jobs:
-  on-success:
+  get-images:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-latest
     environment: InfraK8s
+    steps:
+      - id: get-artifacts
+        name: 'Get artifacts from release workflow'
+        uses: actions/github-script@v6
```

Comment:
I would recommend pinning this to a specific sha instead of a version, we try to do this as a best practice to avoid pulling in a malicious version bump